Card.cs

```diff
@@ -0,0 +1,68 @@
+﻿using EnumColors;
+using System;
+
+namespace Task1_GUI
+{
+    internal class Card : IComparable
+    {
+        private int setterNominal;
+        private Colors.SevenColors setterColor;
+        public int Nominal { get; }
+        public Colors.SevenColors Color { get; }
+        public static int NumOfWinner { get; set; }
+        public Card(string combination)
+        {
+            CheckEnteredCombination(combination);
+            Nominal = setterNominal;
```

Comment:
This is a very very very bad practice. Read about SRP(Single Responsibility Principe). If you name the method `CheckEnteredCombination`, it should only validate the provided value. How can I guess that this method did something else? Use return to load data from this method.